### DISCUSSION TITLE
Metadata Injection

### DISCUSSION THREAD

**Discussion by Qunovaganesh** [apache/hop#4707] [Q&A] [answered]
Dear HOPpers!

For our context, we built around 5-7 pipelines that differ with the transforms within and the order in which these transforms are connected.

Metadata Injection works only if the pipelines are ditto right? And so the concept a of pipeline template.

Please correct me if I'm wrong. 

Thanks in advance.

Regards
Ganesh

**Comment by hansva** [comment, marked as answer]
That is correct.
Metadata injection allows you to use a pipeline template and inject the values for the transforms inside that template.
The template can not be altered. If you want to use it you will have to find a way to generalize your different pipelines into one template.

**Reply by Qunovaganesh** [reply]
Thanks a ton @hansva for clarifying.
